### PR TITLE
Ignore Sigpipe

### PR DESCRIPTION
**Pull request by worktycho**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -472,6 +472,10 @@ int main(int argc, char **argv)
 		#endif  // SIGABRT_COMPAT
 	#endif
 
+
+	#ifdef __unix__
+		std::signal(SIGPIPE, SIG_IGN);
```
Comment (tigerw) [inline src/main.cpp]: You could always set its handler to NonCtrl, and in there do the ignore...?

Comment (worktycho) [inline src/main.cpp]: SIGINT starts a shutdown.

Comment (tigerw) [inline src/main.cpp]: Right, but its action is set to ignore in NonCtrl. For consistency, would you want to do the same for SIGPIPE?

Comment (worktycho) [inline src/main.cpp]: No. SIGINT fires the first time calling NonCtrl. We then initiate a shutdown, and change the handler to ignore future SIGINT signals.

WIth SIGPIPE, we want to ignore all SIGPIPE signals.

Comment (tigerw) [inline src/main.cpp]: Oh, I see. I guess then you could move the "stop" command call, but nevermind.

**Comment by madmaxoft** [comment]
(rebased onto current master)

**Pull request merged**